src/components/DashboardPage/DashboardPage.tsx

```diff
@@ -114,7 +115,49 @@ export const DashboardPage = ({ user, ssrTime, defaultPresetFallback }: External
                         href={routes.project(project.id)}
                         goals={nullable(goals, (g) => (
                             <TreeViewElement>
-                                <GoalTableList goals={g} />
+                                <GoalTableList
+                                    goals={g.map(
+                                        ({
+                                            _shortId,
+                                            _counts,
+                                            _achivedCriteriaWeight,
+                                            title,
+                                            id,
+                                            participants,
+                                            owner,
+                                            tags,
+                                            state,
+                                            updatedAt,
+                                            priority,
+                                            partnershipProjects,
+                                            estimate,
+                                            estimateType,
+                                            projectId,
+                                            project: parent,
+                                        }) => ({
+                                            title,
```

Comment:
don't you want to move it into a function? It would be readable and you could memo it